config/crds/coordination.package-operator.run_adoptions.yaml

```diff
@@ -0,0 +1,213 @@
+                properties:
+                  roundRobin:
+                    description: RoundRobin adoption strategy configuration. Only
+                      present when type=RoundRobin.
+                    properties:
+                      always:
+                        additionalProperties:
+                          type: string
+                        description: Labels to set always, no matter the round robin
+                          choice.
+                        type: object
+                      options:
+                        description: Options for the round robin strategy to choose
+                          from. Only a single label set of all the provided options
+                          will be applied.
+                        items:
+                          additionalProperties:
+                            type: string
+                          type: object
+                        type: array
+                    required:
+                    - always
+                    - options
+                    type: object
+                  static:
+                    description: Static adoption strategy configuration. Only present
+                      when type=Static.
+                    properties:
+                      labels:
+                        additionalProperties:
+                          type: string
+                        description: Labels to set on objects.
+                        type: object
+                    required:
+                    - labels
+                    type: object
+                  type:
+                    default: Static
+                    description: Type of adoption strategy. Can be "Static", "RoundRobin".
+                    enum:
+                    - Static
+                    - RoundRobin
+                    type: string
+                required:
+                - type
+                type: object
+              targetAPI:
+                description: TargetAPI to use for adoption.
+                properties:
+                  group:
+                    type: string
+                  kind:
+                    minLength: 1
+                    type: string
+                  version:
+                    minLength: 1
+                    type: string
+                required:
+                - group
+                - kind
+                - version
+                type: object
+            required:
+            - strategy
+            - targetAPI
+            type: object
+          status:
+            default:
+              phase: Pending
+            description: AdoptionStatus defines the observed state of an Adoption.
+            properties:
+              conditions:
+                description: Conditions is a list of status conditions ths object
+                  is in.
+                items:
+                  description: "Condition contains details for one aspect of the current
```

Comment:
yaml multiline string for readability maybe? I mean, in the end nobody reads this in code .... except that I just did :D